.github/workflows/ci-pep8.yaml

```diff
@@ -14,6 +14,8 @@ jobs:
     steps:
       - name: Checkout 🛎️
         uses: actions/checkout@v2.4.0
+        with:
+          fetch-depth: 0
```

Comment:
sounds unrelated to the pr?

Reply:
Reno needs full git clone to work

Reply:
ok, would have been great to write this in the commit message :)